mopidy/core/playback.py

```diff
@@ -560,3 +572,26 @@ def _trigger_seeked(self, time_position):
         # TODO: Trigger this from audio events?
         logger.debug('Triggering seeked event')
         listener.CoreListener.send('seeked', time_position=time_position)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        return models.PlaybackState(
+            tlid=self.get_current_tlid(),
+            position=self.get_time_position(),
+            state=self.get_state())
+
+    def _restore_state(self, state, coverage):
+        """Internal method for :class:`mopidy.Core`."""
+        if state:
+            new_state = None
+            if 'play-always' in coverage:
+                new_state = PlaybackState.PLAYING
+            if 'play-last' in coverage:
+                new_state = state.state
+            if state.tlid is not None:
+                if PlaybackState.PAUSED == new_state:
```

Comment:
OK. In python unnecessary. 

_'Klugscheiß_modus_an'_ ;-) (sorry don't know the phrase in English, maybe 'smart ass mode on')

That's a hard learned lesson from C programming.
if( value = const )  // Single '=' instead of '=='. Allowed, but not unexpected result. '=' is an assignment, not a comparison operator '==' -> always true if const != 0.

if( const = value) // the compiler error assign-to-const saved the day.

_'Klugscheiß_modus_aus'_